internal/cli/terraform.go

```diff
@@ -82,17 +89,32 @@ func generateTerraformCmdRun(cli *cli, inputs *terraformInputs) func(cmd *cobra.
 			return err
 		}
 
-		if err := generateTerraformConfigFiles(inputs.OutputDIR, data); err != nil {
+		if err := generateTerraformImportConfig(inputs.OutputDIR, data); err != nil {
 			return err
 		}
 
-		cli.renderer.Infof("Terraform config files generated successfully.")
+		if terraformProviderCredentialsAreAvailable() {
+			if err := generateTerraformResourceConfig(cmd.Context(), inputs.OutputDIR); err != nil {
+				return err
+			}
+
+			cli.renderer.Infof("Terraform resource config files generated successfully in: %q", inputs.OutputDIR)
+
+			return nil
+		}
+
+		cli.renderer.Infof("Terraform resource import files generated successfully in: %q", inputs.OutputDIR)
 		cli.renderer.Infof(
 			"Follow this " +
 				"[quickstart](https://registry.terraform.io/providers/auth0/auth0/latest/docs/guides/quickstart) " +
 				"to go through setting up an Auth0 application for the provider to authenticate against and manage " +
 				"resources.",
 		)
+		cli.renderer.Infof(
+			"After setting up the provider credentials, run: "+
+				"`cd %s && terraform init && terraform plan -generate-config-out=generated.tf && terraform apply`",
```

Comment:
Given this is CLI output, would the developer be better served by printing the command in a new line (it's a bit long), and not having backticks around it (in the interests of  both readability and making it easy to copy paste)?

Reply:
Good consideration 👍🏻 updated in [`42d51a4` (#796)](https://github.com/auth0/auth0-cli/pull/796/commits/42d51a456b4a8c0ab13d420d44a707c76932b2c4)